data/init.sql

```diff
@@ -48,6 +48,25 @@ CREATE TABLE IF NOT EXISTS discussion_messages (
 	updated_at TIMESTAMP
 );
 
+<<<<<<< HEAD
+
```

Comment:
what's this :o

Reply:
its also not in my file, this is so weird, what have I done ?